### PR TITLE
Spc 442 camera processor readme documentation

### PR DESCRIPTION
**Pull request by Sxord**

### CODE REVIEW

File: CameraProcessor/README.md [hunk 1]
```diff
@@ -1,11 +1,74 @@
...
+
+## Architecture
+
+The inner workings of the camera processor consist of three parts. 
+The input that is processed by the pipeline and send to either other camera processors or the interface.
+
+### Input
+
+There are two inputs available to the camera processor, 
+the unaltered video stream and commands/feature maps send by the orchestrator.
```
Comment (Marieke-van-der-Plaat) [inline CameraProcessor/README.md:17]: Sent

File: CameraProcessor/README.md [hunk 2]
```diff
@@ -1,11 +1,74 @@
...
+The job of the camera processor is to detect all potential subjects 
+and track the subjects that have been selected by the camera operators using the Interface.
+The camera operators only see the detections within a frame, and the subjects they themselves started tracking.
+The tracking of subjects is performed over multiple camera's, 
+notifying the camera operator when a subject is re-identified.
+
+## Architecture
+
+The inner workings of the camera processor consist of three parts. 
+The input that is processed by the pipeline and send to either other camera processors or the interface.
```
Comment (Marieke-van-der-Plaat) [inline CameraProcessor/README.md:12]: Sent

File: CameraProcessor/processor/webhosting/README.md [hunk 3]
```diff
@@ -0,0 +1,57 @@
+# Webhosting
+
+The content of this folder uses the [Tornado](https://www.tornadoweb.org/en/stable/) library for the communication with the outside world.
+There are two ways we use Tornado.
+
+One creates a websocket connection with the Processor Orchestrator component.
+This is used during deployment of the Camera Processor.
```
Comment (Marieke-van-der-Plaat) [inline CameraProcessor/processor/webhosting/README.md:7]: the deployment

File: CameraProcessor/processor/webhosting/README.md [hunk 4]
```diff
@@ -0,0 +1,57 @@
...
+- [websocket_client.py](websocket_client.py): Defines the websocket logic
+- [start_command.py](start_command.py): Start command object
+- [stop_command.py](stop_command.py): Stop command object
+
+The other method is for docker debugging and remote verification purposes.
+This creates a http server which can be connected to see the output of a single Camera Processor online without having to boot up the entire system.
```
Comment (Marieke-van-der-Plaat) [inline CameraProcessor/processor/webhosting/README.md:13]: This  creates AN http server

Comment (Marieke-van-der-Plaat) [inline CameraProcessor/processor/webhosting/README.md:13]: Which can vervangen door that can

File: CameraProcessor/processor/webhosting/README.md [hunk 5]
```diff
@@ -0,0 +1,57 @@
...
+- [html_page_handler.py](html_page_handler.py): Serves the webpage in which the stream is visible
+- [stream_handler.py](stream_handler.py): Serves the individual frames to the webclient
+
+### Websockets
+The [WebsocketClient](websocket_client.py) creates a websocket connection with the Processor Orchestrator component.
+Given a websocket url the [WebsocketClient](websocket_client.py) will try to connect for a minute.
```
Comment (Marieke-van-der-Plaat) [inline CameraProcessor/processor/webhosting/README.md:19]: Komma na url

**Review by Marieke-van-der-Plaat — DISMISSED** [review]

**Review by saberzero1 — DISMISSED** [review]

File: CameraProcessor/README.md [hunk 6]
```diff
@@ -15,32 +79,28 @@ Dependencies install
 pip install -r requirements.txt
```
Comment (GerardvSchie) [inline CameraProcessor/README.md:79]: This must be requirements-gpu.txt

**Review by GerardvSchie — CHANGES_REQUESTED** [review]
It misses a section explaining the config file
what are things that a user might want to change

For example the HLS enabled and the mode are quite important

File: CameraProcessor/README.md (hunk 6 above)
Comment (saberzero1) [reply CameraProcessor/README.md]: `requirements-gpu.txt` has been fixed.
Config will still need to be explained.

**Review by saberzero1 — APPROVED** [review]

**Review by GerardvSchie — APPROVED** [review]

**Pull request merged**